core-plugins/src/e2e-test/java/io/cdap/plugin/common/stepsdesign/CorePlugin.java

```diff
@@ -102,4 +104,38 @@ private void readInputStream(InputStream input, List<String> lst) throws IOExcep
       }
     }
   }
+
+  @Then("Validate output records in output folder path {string} is equal to expected output file {string}")
+  public void validateOutputRecordsInOutputFolderIsEqualToExpectedOutputFile(String outputFolder
+    , String expectedOutputFile) throws IOException {
+
+    StringBuilder expectedOutput = new StringBuilder(StringUtils.EMPTY);
```

Comment:
Make expectedOutput a list of lines instead of a string containing concatenated lines.

Reply:
Change done.